src/pages/components/filter-panel.mdx

```diff
@@ -0,0 +1,276 @@
+
+<Caption>Example of parent tier filter item selected, with child tier items exposed</Caption>
+
+</Column>
+
+</Row>
+
+### Multiselect
+Use a multiselect filter when the user has the option to pick more than one attribute to modify the data results. Under the hood, multiselects behave like checkboxes.
+
+<Row>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: multiselect with no selected filter item.](../../images/component/filter-panel/filter-panel-multiselect-not-selected.jpg)
+
+<Caption>Example of multiselect with no selected filter item</Caption>
+
+</Column>
+
+<Column colMd={4} colLg={4}>
+
+![Example image: multiselect with multiple filter items selected.](../../images/component/filter-panel/filter-panel-multiselect-selected.jpg)
+
+<Caption>Example of multiselect with multiple filter items selected</Caption>
+
+</Column>
+
+</Row>
+
+### Multiple categories
+A category is a set of filter items within the same topic. For example, “size” is a category and small, medium, large, and extra large are its filter choices. Selections from multiple categories may be applied to the same data set. For example, the user can filter by size, color, and price range.
+
+![A visual example of the results being filtered by selections from within two categories](../../images/component/filter-panel/filter-panel-multiple-categories.jpg)
+
+<Caption>Example of results being filtered by selections from within two categories</Caption>
+
+## Behaviors
+
+### Expanding and collapsing
+The filter panel can be configured with any number of filters and categories. 
+
+* The user may expand multiple filter categories at a time.
+* The user may collapse all categories.
+* The user can collapse categories that have filters selected. 
+
+### Larger breakpoints (lg, xl, max)
+The filter panel spans 4 columns of the 16 column grid with a customizable height. 
+
+#### Scroll
+Filter panel scrolls with the page. Depending on the amount of filters available to the user, the filter panel could exceed the browser height.
+
+#### Keyboard interaction
+Filter panel includes a Skip to main content option for keyboard users so they can jump into the main content and/or search results.
+
+### Smaller breakpoints (sm and md)
+On smaller breakpoints, the filter panel is triggered by a button that can be configured anywhere within the experience. Once the user selects the filter button, a modal will open and display the filter panel, the user can close the modal to view the filtered results. The filter button should indicate how many filters the user has selected.
+
+![A visual example of the Filter panel component at a smaller breakpoint](../../images/component/filter-panel/filter-panel-behaviors-smaller-breakpoints.jpg)
+
+<Caption>Example of the filter panel in smaller breakpoints. The user selects the filter button, selects a number of filter items, and closes the modal to view the results.</Caption>
```

Comment:
I think it might be good to mention something like "From left to right, the mobile screens above shows these states: no filter applied, filter panel open, and filters active." Key word "left to right". Shouldn't assume people from all cultures will have the same sense of progression.

Reply:
@shixiedesign Thanks for the feedback, makes absolute sense! I updated the caption but didn't say "mobile" because we don't usually reference small and medium as mobile and the section is called "smaller breakpoints". 

LMK what you think!